File: punto1parcinume.py
def medidadetiempo(matriz, tama):
	inicio = time.process_time()
	matri(matriz, tama)
	fin = time.process_time( )
	return float(fin - inicio)


def matri(matriz, tama):
	for i in range (tama):
		for j in range (tama):
			print("ingrese los valores de la matriz")
			matriz[i][j] = input()
	for i in range (tama):
		for j in range (tama):
			print(matriz[i][j])
	print("\n")
	suma=0
	for i in range (tama):
		for j in range(tama):
			if(j>i):
				suma+=matriz[i][j]
		print( "La suma de los elementos de la submatriz triangular superior es : ", suma)



print ("digite el tamaño de las filas y las columnas de la matriz")
tama=input()
medidadetiempo(matriz,tama)

File: punto1parcinume.c
void main(){
int matriz[][];
int i,j,n=3,suma=0,triangu_sup=1;
	for(i=0;i<n-1;i++)
	{
		for(j=n-1;j>0;j--)
		{
			if(matriz[i][j] == 0){
				suma = suma + matriz[i][j];
				triangu_sup *=1;
			}
			else
				triangu_sup *=0;
		}	
	}
}
